### PR TITLE
#11359 Switch the toolbar back to standard view when disabling the da…

### PR DESCRIPTION
**Pull request by ViktorGraz**
…rk mode
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11359, https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10544, https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10212

### CODE REVIEW

File: PowerEditor/src/WinControls/Preference/preference_rc.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 	#define	PREF_MSG_ISCHECKED_GENERALPAGE (WM_USER + 1) // wParam:checkbox/radiobutton ID in General page. lParam is type of "bool *" to get result
 	#define	PREF_MSG_SETTOOLICONSFROMSTDTOSMALL (WM_USER + 2) 
 	#define	PREF_MSG_DISABLETABBARALTERNATEICONS (WM_USER + 3) 
+	#define PREF_MSG_SETTOOLICONSFROMSMALLTOSTD (WM_USER + 4)
```
Comment (Mateos81) [inline PowerEditor/src/WinControls/Preference/preference_rc.h:23]: Small remark: the previous entries have a tab between the #define keyword and the constant name, while yours has a space in-between

Comment (ViktorGraz) [reply PowerEditor/src/WinControls/Preference/preference_rc.h]: @Mateos81, thanks for the review, I have changed it!

**Comment by ArkadiuszMichalski** [comment]
@ViktorGraz You always back to `Standard icon:small`, even if someone have choosed other sets before switch `Dark mode`. This is probably not the expected behavior. Also after turn off `Dark mode` we have this state in `Preferences`:
image

**Comment by ViktorGraz** [comment]
@ArkadiuszMichalski , thanks for the review!

**Pull request closed without merging**